test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/RemoteAttributeTest.cs

```diff
@@ -5,32 +5,24 @@
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc.Abstractions;
-using Microsoft.AspNetCore.Mvc.DataAnnotations;
 using Microsoft.AspNetCore.Mvc.ModelBinding;
 using Microsoft.AspNetCore.Mvc.ModelBinding.Validation;
 using Microsoft.AspNetCore.Mvc.Routing;
 using Microsoft.AspNetCore.Mvc.ViewFeatures;
 using Microsoft.AspNetCore.Routing;
 using Microsoft.AspNetCore.Testing;
 using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Localization;
 using Microsoft.Extensions.Logging;
 using Microsoft.Extensions.Logging.Abstractions;
 using Microsoft.Extensions.ObjectPool;
 using Microsoft.Extensions.Options;
 using Moq;
 using Xunit;
-using Resources = Microsoft.AspNetCore.Mvc.ViewFeatures.Test.Resources;
 
 namespace Microsoft.AspNetCore.Mvc
 {
     public class RemoteAttributeTest
```

Comment:
This test class doesn't overlap with `PageRemoteAttributeTest` as much as I expected:
- Should we add specific tests of `GetUrl(...)` to this class?
- Should we add more tests covering the behaviour of `GetUrl(...)` when properties are not set to `PageRemoteAttributeTest`?
- Is `RemoteAttributeBaseTest` missing a few tests that could be migrated from `RemoteAttributeTest`?

I'm not positive further changes are needed but would like us to think about it before calling this PR done. @pranavkm what are your thoughts in this area?